rubocop-bsm.gemspec

```diff
@@ -17,9 +17,9 @@ Gem::Specification.new do |spec|
 
   spec.add_runtime_dependency 'rubocop', '~> 1.0'
   spec.add_runtime_dependency 'rubocop-performance'
-  spec.add_runtime_dependency 'rubocop-rails'
   spec.add_runtime_dependency 'rubocop-rake'
   spec.add_runtime_dependency 'rubocop-rspec'
 
   spec.add_development_dependency 'rake'
+  spec.add_development_dependency 'rubocop-rails'
```

Comment:
rubocop-rails comes with activesupport as a primary dependency. I prefer not to include it by default